components/org.wso2.carbon.identity.auth.attribute.handler/src/main/java/org/wso2/carbon/identity/auth/attribute/handler/exception/AuthAttributeHandlerException.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.auth.attribute.handler.exception;
+
+import org.wso2.carbon.identity.base.IdentityException;
+
+/**
+ * Exception class that represents exceptions thrown from Auth Attribute Handler component.
+ */
+public class AuthAttributeHandlerException extends IdentityException {
+
+    private static final long serialVersionUID = -5419168766998504972L;
+
+    public AuthAttributeHandlerException(String message) {
```

Comment:
Do we need to provide this constructor since it does not have an error code param?

Reply:
removed